Semantic episode (#54)
4TUResearchData-Carpentries/interoperability-climate-sciences: episodes/semantic-interoperability.md

---
title: "Semantic interoperability"
teaching: 30
exercises: 20
---

:::::::::::::::::::::::::::::::::::::: questions

* What is semantic interoperability, and which kinds of meaning must be made explicit?
* Why can two structurally similar datasets still be scientifically incompatible?
* What is the difference between a label, a controlled vocabulary, a code list, and an ontology?
* Where can researchers discover, evaluate, and share semantic artefacts for the Earth sciences?
* How do the CF Conventions encode the meaning and context of climate and atmospheric data?
* Is using the same CF `standard_name` sufficient to make two variables directly comparable?
* What does it mean for a NetCDF file to conform to a particular version of the CF Conventions?

> * What can—and what cannot—a CF compliance checker establish?

Reviewer: ```suggestion
* What can a CF compliance checker determine, and what are its limitations?
```
